<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GameWidget</class>
 <widget class="QWidget" name="GameWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>700</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" stretch="0,1">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QFrame" name="statsFrame">
     <property name="styleSheet">
      <string notr="true">#statsFrame {
    background-color: #333;
    border-bottom: 1px solid #555;
}
QLabel {
    color: white;
    font-weight: bold;
    padding: 5px;
}
QProgressBar {
    text-align: center;
    color: white;
    font-weight: bold;
    border: 1px solid #555;
    border-radius: 5px;
}
QProgressBar::chunk {
    background-color: #4CAF50;
    border-radius: 4px;
}</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>‚ù§Ô∏è ÁîüÂëΩ:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QProgressBar" name="healthBar">
        <property name="maximumSize">
         <size>
          <width>200</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="value">
         <number>100</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="scoreLabel">
        <property name="text">
         <string>üèÜ ÂàÜÊï∞: 0</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="waveLabel">
        <property name="text">
         <string>üåä Ê≥¢Ê¨°: 1</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="enemyCountLabel">
        <property name="text">
         <string>üëæ Êïå‰∫∫: 0</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="fpsLabel">
        <property name="text">
         <string>FPS: 0</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGraphicsView" name="gameView">
     <property name="styleSheet">
      <string notr="true">border: none; background-color: #1e1e1e;</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
